CONTRIBUTING.md

```diff
@@ -224,10 +224,13 @@ To generate the experimental TypeScript version for a particular release, run:
 
 ```sh
 git checkout v4.X.X
```

Comment:
This is not needed now, right? Besides, it'll fail if the `v4.X.X` branch doesn't previously exist (and it shouldn't exist).

Reply:
done https://github.com/algolia/instantsearch.js/pull/4307/commits/124555ee339060f7faa7d907df23fcb142e254b2